components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -961,6 +962,12 @@ subroutine ocn_run_mct( EClock, cdata_o, x2o_o, o2x_o)!{{{
             call mpas_timer_stop("land_ice_build_arrays")
 
             call ocn_frazil_forcing_build_arrays(domain, meshPool, forcingPool, diagnosticsPool, statePool, ierr)
+
+            ! Compute normalGMBolusVelocity, relativeSlope and RediDiffVertCoef if respective flags are turned on
+            if (config_use_Redi.or.config_use_GM) then
+              call ocn_gm_compute_Bolus_velocity(statePool, diagnosticsPool, meshPool, scratchPool, timeLevelIn=1)
```

Comment:
@vanroekel, the addition of this line is causing an exact restart test to fail. Why is this needed outside the normal `ocn_timestep` call, rather than within? 

Also, when this line was missing, all the ERS tests passed (i.e. Monday's testing). But that is meaningless, because all the diagnostic arrays for Redi and GM simply were not computed at all. Do you agree?

Reply:
Yes, that is why everything passed.   GM was never called.  This needs to be at the start of a timestep instead of the end for restart for restart to pass.  This passed MPAS restart, I'm really confused why this fails now.  In mpas this is in the mode_forward code, in a similar place.  So I don't understand why this isn't passing.

It can be inside timestep but seemed easier to call where I put it as we were already loading the pools and it seemed cleaner than in the timestep code, but as long as it is before the timestep calculation (ocn_tendency) this is fine.